### 2. To collect data in an introductory statistics course, a colleague recently gave his students a questionnaire. One questions asked whether the student was a vegetarian. Of 25 students, none answered "yes."The group was not a random sample of students (they had selected this class), but we will use these data to illustrate inference for a proportion. Let $\pi$ represent the population proportion who would say "yes." Consider $H_{0}$:0.50 and  and $H_{a}$ $\ne$ 0.5

#### A. What happens when you try to use the Wald test in which the z-statistic uses the estimated standard error? 

$H_{0}$:p=0.50 and $H_{a}:$p $\ne$ 0.5

SE =$(\frac{\hat{p}(1-\hat{p})}{n})^{1/2}$

Z= $\hat{p}$- $p_{0}$/SE

In [37]:
p_hat<- 0/25
p_hypothesized<-.5
n<- 25
SE<-sqrt(p_hat*(1-p_hat)/n)
Wald_test<-(p_hat-p_hypothesized)/SE
Wald_test
SE
p_hat*(1-p_hat)

[1] -Inf

[1] 0

[1] 0

When I try to use the Wald test in which the z-statistic uses the esimated standard error I get is undefined. That is due to the z statistic = 0/0.

#### B. Find the 95% Wald. conf. interval for $\pi$. Is it believable? 

In [2]:
install.packages("binom")
library(binom)
w<-0
n<- 25
alpha<-0.05
binom.confint(x=w,n=n,conf.level = 1 - alpha, methods ="asymptotic")

Updating HTML index of packages in '.Library'
Making 'packages.html' ... done


method     x n  mean lower upper
1 asymptotic 0 25 0    0     0

A confidence interval exist but that is not believable because with SE=0, that means $\hat{p}=0 +/- 0 which is still 0.

#### C. Conduct the "score" test in which the z-statistic uses the null hypothesis of $\pi$ in the standard error. Report and interpret the p-value for this test. 

$H_{0}: p=0.50$

$H_{a}:p\ne 0.50$

SE = $(\frac{p_{0}(1-p_{0})}{n})^{1/2}$

Z= $\hat{p}$- $p_{0}$/SE

In [31]:
p_hat<-0/25
p_hypothesized<-0.50
n<- 25
SE<-sqrt(p_hypothesized*(1-p_hypothesized)/n)
Score_test<-(p_hat-p_hypothesized)/SE
Score_test
prop.test(0,25,p=0.5, alternative="two.sided",0.95,correct=FALSE)

[1] -5


	1-sample proportions test without continuity correction

data:  0 out of 25, null probability 0.5
X-squared = 25, df = 1, p-value = 5.733e-07
alternative hypothesis: true p is not equal to 0.5
95 percent confidence interval:
 0.0000000 0.1331923
sample estimates:
p 
0 


The p-value associated with this "score" test is 5.733e-07 which is < .05, it appears that we have sufficient evidence such that we can reject the null hypothesis that the true proportion of students in the class who would say yes to being vegetarian is equal to 0.5
The results are unusual and should be pursued further. 

#### D. Compute the 95% score CI for $\pi$. This requires you to solve the quadratic equation |Z|<1.96, $\pi$ is the unknown variable in this equation.

In [4]:
w<-0
n<- 25
alpha<-0.05
binom.confint(x=w,n=n,conf.level = 1 - alpha, methods ="wilson")

method x n  mean lower upper    
1 wilson 0 25 0    0     0.1331923

Wilson Score Interval: We are 95% confident that the true proportion of students in the class who would say yes to being vegetarian is between 0 and 0.13 